core/src/main/java/discord4j/core/object/ExtendedInvite.java

```diff
@@ -69,10 +69,17 @@ public int getMaxUses() {
      * @return The instant this invite expires, if possible.
      */
     public Optional<Instant> getExpiration() {
```

Comment:
Well, since `Optional` is always present here, then why not return `Instant` itself?

Reply:
i have keept the optional instant, as invites can be infinite (never expiring), the value is then set to zero, like the total usages.
In this case an empty optional is returned.

Reply:
Thanks for the clarification